shiksha-website/shiksha-frontend/src/app/view/user/question-bank/question-bank-generation/question-bank-generation.component.html

```
@@ -274,21 +274,21 @@ <h2 class="text-base font-bold text-content mb-3">Topic-wise Marks Distribution<
                     </thead>
                     <tbody>
                       <tr *ngFor="let c of marksDistribution; let i=index" class="hover:bg-gray-50">
-                        <td class="p-3 border text-sm font-medium">{{c.unit_name}}</td>
+                        <td class="p-3 border text-sm font-medium">{{c.unitName}}</td>
                         <td class="p-3 border">
                           <input type="text" class="w-full text-center border-none p-1 focus:ring-0"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)